### PR TITLE
sci-hub.cc

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Sci-hub.cc.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="Sci-hub.cc">
+	<target host="sci-hub.cc" />
+	<target host="www.sci-hub.cc" />
+	<target host="sci-hub.ac" />
+	<target host="www.sci-hub.ac" />
+
+	<rule from="^http://(?:www\.)?sci-hub\.(?:cc|ac)/" to="https://sci-hub.ac/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Sci-hub.cc.xml]: Please document why and remove non-capturing group.

**Comment by J0WI** [comment]
It seems that several documents live on their own subdomain (where https is refused). Could you document this behavior?

**Comment by galeksandrp** [comment]
Done.

**Comment by J0WI** [comment]
I meant a generic hint, that the mirrors of the documents live on their own subdomain where https is refused, because there might be plenty of them.

**Comment by galeksandrp** [comment]
Documented.

**Comment by J0WI** [comment]
It seems that some aliases are currently down.

**Comment by galeksandrp** [comment]
Documented.

**Pull request merged**